### PR TITLE
add moreutils

### PR DESCRIPTION
**Pull request by matthiasdiener**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by matthiasdiener** [comment]
@conda-forge/staged-recipes this is ready for review.

File: recipes/moreutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+
+  run:
+    - perl
```
Comment (jakirkham) [inline recipes/moreutils/meta.yaml:22]: Should `perl` be in `host` as well?

File: recipes/moreutils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  commands:
+    - echo "foo" | ts
```
Comment (jakirkham) [inline recipes/moreutils/meta.yaml:26]: Should the other tools included be tested as well?

**Review by jakirkham — COMMENTED** [review]
Thanks for the PR Matthias! 😄

Had a couple questions below :)

File: recipes/moreutils/meta.yaml (hunk 1 above)
Comment (matthiasdiener) [reply recipes/moreutils/meta.yaml:22]: I'm not sure, `perl` does not seem necessary to build the package.

File: recipes/moreutils/meta.yaml (hunk 2 above)
Comment (matthiasdiener) [reply recipes/moreutils/meta.yaml:26]: There isn't really a test suite for this package, but I added another small test.

**Comment by matthiasdiener** [comment]
Pinging @conda-forge/staged-recipes for review.

**Pull request merged**